scripts/toml_to_moha.py

```diff
@@ -0,0 +1,170 @@
+import tomllib
+import numpy as np
+from os.path import exists
+from os import makedirs
+import sys
+sys.path.insert(0, '../')
+import moha
+
+def set_defaults(input_data):
+    '''
+    Function for setting defaults in the parameter dictionary
+    loaded from toml input file.
+
+    Parameters
+        ----------
+        input_data: dict
+            dict loaded from toml input file.
+
+        Returns
+        -------
+        None
+    '''
+
+    # require that defaults.toml exists
+    required_default_paramfile = "defaults.toml"
+    if not exists(required_default_paramfile):
+        raise Exception("Default input file 'defaults.toml' is required.")
+    
+    # load defaults.toml data into default_data
+    with open(required_default_paramfile, "rb") as f:
```

Comment:
we can rewrite it in one line, that is still easy to read. I propose to have it done as 
```
default_data = tomlib.load(
                       open(required_default_paramfile, "rb")
            )
```
The advantage of this, that we don't need to open and then close file explicitly